package.json

```diff
@@ -56,15 +60,11 @@
       "coverage/**",
       "src/comm/*.js",
       "src/comm/client/**",
-      "src/comm/rate-control/pidRate.js",
-      "src/comm/rate-control/rateControl.js",
-      "src/comm/rate-control/rateInterface.js",
       "src/composer/**",
       "src/contract/**",
       "src/fabric/**",
       "src/gui/**",
       "src/iroha/**",
-      "src/sawtooth/**",
```

Comment:
I could not tell if there was a reason for all of these excludes or not. I left most. The rate-control files above actually already have some coverage.